README.md

```diff
@@ -441,7 +441,7 @@ If you use Mava in your work, please cite the accompanying
 [roadmap]: https://github.com/instadeepai/Mava/issues/246
 [wishlist]: https://github.com/instadeepai/Mava/issues/247
 [bsuite]: https://github.com/deepmind/bsuite
-[quickstart]: https://github.com/instadeepai/Mava/blob/develop/examples/tf/quickstart.ipynb
+[quickstart]: https://github.com/instadeepai/Mava/blob/develop/examples/jax/quickstart.ipynb
```

Comment:
Before we deprecate TF, can we please keep both quickstart notebooks? That way they are easy to mention in the docs.

Reply:
Sure. Currently, the other notebook is still available in Mava. It is just not referenced in the readme. Should I keep a reference to both notebooks in the readme as well?

Reply:
It is a weird in between space that we are in since we have not made a JAX release so difficult answer. I guess we could reference both till TF deprecated?